client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -118,9 +120,21 @@ object CapiModelEnrichment {
         "society/series/this-is-the-nhs"
       )
 
+      val hashedSpecialReportTags: Set[String] = Set(
+        "a22340dd6b783a7fa481fa812f8f845e"
```

Comment:
Is it worth documenting somewhere how this can be generated if needed?

Reply:
Good point.  I've added a comment showing how to generate them.